priv/data/community/blogs.exs

```diff
@@ -0,0 +1,82 @@
+[
+  %{
+    name: "Dashbit",
+    link: "https://dashbit.co/blog/",
+    icon: "/images/community/blogs/dashbit.png",
+    logo: nil,
+    about:
+      "Maintainers and contributors to many Elixir projects, such as Ecto, Broadway, LiveView, and Elixir itself."
+  },
+  %{
+    name: "OTP Team",
+    link: "https://blog.erlang.org/",
+    icon: "/images/community/blogs/otp.png",
+    logo: nil,
+    about:
+      "The Erlang/OTP team at Ericsson, the implementors and maintainers of Erlang/OTP."
+  },
+  %{
+    name: "Elixir Lang",
+    link: "https://elixir-lang.org/blog/",
+    icon: "/images/community/blogs/elixir.png",
+    logo: nil,
+    about:
+      "Official blog of the Elixir programming language."
+  },
+  %{
+    name: "Gleam News",
+    link: "https://gleam.run/news/",
+    icon: "/images/community/languages/gleam.svg",
+    logo: nil,
+    about:
+      "Latest official news on the Gleam programming language."
+  },
+  %{
+    name: "LFE blog",
+    link: "https://blog.lfe.io/",
+    icon: "/images/community/blogs/lfe.png",
+    logo: nil,
+    about:
+      "Official blog of the LFE programming language."
+  },
+  %{
+    name: "Planet Erlang",
+    link: "http://planeterlang.com/",
+    icon: nil,
+    logo: nil,
+    about:
+      "Planet Erlang is a meta blog that collects posts from the blogs of various Erlang developers and contributors."
+  },
+  %{
+    name: "Elixir Radar",
+    link: "https://elixir-radar.com/",
+    icon: nil,
+    logo: nil,
```

Comment:
Do you need our logo?

Unfortunately, we don't an icon. :(

Reply:
If you have a logo that we could use here, that'd be great! I looked around for icons but as you say, didn't find any :) I think the current "logos" are used the same way Elixir's logo is used here: https://erlef.org/community/.

As an aside to @starbelly,  I think this use of logos will probably need some extra love at some point. We'll want to ensure the names don't need to be removed (as in Elixir's case) for better accessibility and SEO :) I can look at that in a separate PR.

Reply:
Logo is fine. We merely make a distinction (which is fuzzy) between icon and logo for css classes iirc. In this sub-domain an icon is said to be an image without words, a logo is an image with words. In reality of course the distinction between logo and icon is not exactly that, but it helps here :)

Reply:
@joaquinalcerro I agree 💯 . This app needs ❤️ all over the place! 😁

Reply:
Hi @starbelly sure thing, I will check the open issues to try addressing some of them. You guys have done a amazing job so far...